handsontable/src/plugins/exportFile/exportFile.js

```diff
@@ -111,6 +111,7 @@ export class ExportFile extends BasePlugin {
    * @property {boolean} [rowHeaders=false] Include row headers in the exported file.
    * @property {string} [columnDelimiter=','] Column delimiter.
    * @property {string} [range=[]] Cell range that will be exported to file.
+   * @property {boolean|RegExp|Function} [sanitizeValues=false] Controls the sanitization of cell value.
    */
```

Comment:
replace Function with callback https://jsdoc.app/tags-callback 

```suggestion
   * @property {boolean|RegExp|sanitizeCallback} [sanitizeValues=false] Controls the sanitization of cell value.
   */
   
   /**
   * This callback is displayed as a global member.
   * @callback sanitize callback
   * @param {string} rawInput 
   * @return {string} sanitizeValues 
   */
```

Reply:
I'd leave it as well for now tbh,
1. I don't think we ever used it and it would require testing with the documentation to see how it's rendered
2. This callback is as simple as they get - it receives 1 argument and returns it after processing, so it's not really that confusing.